code/datums/looping_sounds/weather.dm

```diff
@@ -44,4 +44,33 @@
 	start_sound = 'sound/weather/ashstorm/inside/weak_start.ogg'
 	start_length = 130
 	end_sound = 'sound/weather/ashstorm/inside/weak_end.ogg'
+	volume = 40
+
+/datum/looping_sound/weather/wind
+	mid_sounds = list(
```

Comment:
Why is this list not static

Reply:
It's an already existing system

Reply:
And? You're expanding it, alot.

Reply:
No, I'm not touching the system at all. I'm adding more type definitions. I can add a cache for those lists too if you want.

Reply:
Aye, please.